src/components/pages/AllMoviesPage.vue

```diff
@@ -0,0 +1,116 @@
+<script>
+import { defineComponent } from 'vue';
+import { mapState } from "pinia";
+import movies from '@/stores/moviesStore';
+import BreadCrumbs from '@/components/common/BreadCrumbs.vue';
+import SectionTitle from '@/components/common/Section/SectionTitle.vue';
+import AppSelect from '@/components/common/App/AppSelect.vue';
+import MovieCard from '@/components/views/MovieCard.vue';
+import AppInput from '@/components/common/App/AppInput.vue';
+import AppLabel from '@/components/common/App/AppLabel.vue';
+import TheContainer from '@/components/common/TheContainer.vue';
+
+export default defineComponent({
+  components: { BreadCrumbs, SectionTitle, AppSelect, MovieCard, AppInput, AppLabel, TheContainer },
+  data() {
+    return {
+      optionSelected: 'All categories',
+      inputContent: '',
+    }
+  },
+  computed: {
+    ...mapState(movies, ["movies"]),
+    moviesCategories() {
+      const allCategories = this.movies.map(movie => movie.genre.name)
+      return new Set(allCategories)
+    },
```

Comment:
Considering we rely only on store data itself anyway... why not make this one a getter in the store itself? 👀

Reply:
Good point! I will move it to the store. 👍